src/lang/std/sketch.ts

```diff
@@ -100,7 +100,7 @@ import type { EdgeCutInfo } from '@src/machines/modelingMachine'
 const STRAIGHT_SEGMENT_ERR = new Error(
   'Invalid input, expected "straight-segment"'
 )
-const ARC_SEGMENT_ERR = new Error('Invalid input, expected "arc-segment"')
+const ARC_SEGMENT_ERR = () => new Error('Invalid input, expected "arc-segment"')
```

Comment:
It's really hard to trace where the error was returned when the same error is used everywhere, making this a function fixes that.